config/routes.rb

```diff
@@ -1,5 +1,13 @@
+require 'sidekiq/web'
+
+Sidekiq::Web.use Rack::Auth::Basic do |username, password|
+  username == 'mtom' && password == 'basic elephantitis swagger'
```

Comment:
We should use environment variables or a config file here, given that the source is publicly available.